elastic/kibana
# [TableListView] Hide tag filter when savedObjectTagging is not provided

In this PR I removed the tags filter dropdown when consumer don't provide the `saveObjectTagging` dependency.

If they don't it means that their content does not use tagging (e.g. files).

## How to test

* Upload a file (I use the Security > Cases, create a case and then attach a file)
* Navigate to Stack management > Kibana > File
* Check that the table does not render the "tags" filter

Fixes https://github.com/elastic/kibana/issues/162257
